### PR TITLE
Move 14 more functions from salt.utils

### PR DESCRIPTION
**Pull request by terminalmage**
salt.utils.test_mode -> salt.utils.args.test_mode
salt.utils.split_input -> salt.utils.args.split_input
alt.utils.pem_finger -> salt.utils.crypt.pem_finger
salt.utils.is_bin_file -> salt.utils.files.is_binary
salt.utils.list_files -> salt.utils.files.list_files
salt.utils.safe_walk -> salt.utils.files.safe_walk
salt.utils.st_mode_to_octal -> salt.utils.files.st_mode_to_octal
salt.utils.normalize_mode -> salt.utils.files.normalize_mode
salt.utils.human_size_to_bytes -> salt.utils.files.human_size_to_bytes
salt.utils.get_hash -> salt.utils.hashutils.get_hash
salt.utils.is_list -> salt.utils.data.is_list
salt.utils.is_iter -> salt.utils.data.is_iter
salt.utils.isorted -> salt.utils.data.sorted_ignorecase
salt.utils.is_true -> salt.utils.data.is_true

### CODE REVIEW

File: salt/key.py [hunk 1]
```diff
@@ -22,8 +22,10 @@
 import salt.daemons.masterapi
 import salt.exceptions
 import salt.minion
-import salt.utils
+import salt.utils  # Can be removed once get_master_key is moved
```
Comment (cachedout) [inline salt/key.py]: If you use the `TODO` keyword here, we can have CodeClimate track it.

File: salt/modules/ebuild.py [hunk 2]
```diff
@@ -21,7 +21,7 @@
 import re
 
 # Import salt libs
-import salt.utils
+import salt.utils  # Can be removed once alias_function is moved
```
Comment (cachedout) [inline salt/modules/ebuild.py]: Same comment here wrt `TODO`

File: salt/modules/freebsdpkg.py [hunk 3]
```diff
@@ -80,7 +80,7 @@
 import re
 
 # Import salt libs
-import salt.utils
+import salt.utils  # Can be removed when alias_function is moved
```
Comment (cachedout) [inline salt/modules/freebsdpkg.py]: Another `TODO`

File: salt/modules/mac_brew.py [hunk 4]
```diff
@@ -17,7 +17,7 @@
 import logging
 
 # Import salt libs
-import salt.utils  # Can be removed when alias_function, is_true are moved
+import salt.utils  # Can be removed when alias_function is moved
```
Comment (cachedout) [inline salt/modules/mac_brew.py]: `TODO`

File: salt/modules/mac_ports.py [hunk 5]
```diff
@@ -37,7 +37,7 @@
 import re
 
 # Import salt libs
-import salt.utils  # Can be removed when alias_function, is_true are removed
+import salt.utils  # Can be removed when alias_function is removed
```
Comment (cachedout) [inline salt/modules/mac_ports.py]: `TODO`

File: salt/modules/pacman.py [hunk 6]
```diff
@@ -18,7 +18,7 @@
 import os.path
 
 # Import salt libs
-import salt.utils
+import salt.utils  # Can be removed once alias_function, fnmatch_multiple are moved
```
Comment (cachedout) [inline salt/modules/pacman.py]: `TODO`

File: salt/modules/pkgutil.py [hunk 7]
```diff
@@ -14,7 +14,7 @@
 import copy
 
 # Import salt libs
-import salt.utils  # Can be removed once alias_function, is_true are moved
+import salt.utils  # Can be removed once alias_function is moved
```
Comment (cachedout) [inline salt/modules/pkgutil.py]: `TODO`

File: salt/modules/solarisips.py [hunk 8]
```diff
@@ -43,7 +43,7 @@
 
 
 # Import salt libs
-import salt.utils  # Can be removed once alias_function, is_true are moved
+import salt.utils  # Can be removed once alias_function is moved
```
Comment (cachedout) [inline salt/modules/solarisips.py]: `TODO`

File: salt/modules/solarispkg.py [hunk 9]
```diff
@@ -16,7 +16,7 @@
 import logging
 
 # Import salt libs
-import salt.utils  # Can be removed once alias_function, is_true are moved
+import salt.utils  # Can be removed once alias_function is moved
```
Comment (cachedout) [inline salt/modules/solarispkg.py]: `TODO`

File: salt/modules/state.py [hunk 10]
```diff
@@ -28,10 +28,12 @@
 import salt.config
 import salt.payload
 import salt.state
-import salt.utils
+import salt.utils  # Can be removed once namespaced_function is moved
```
Comment (cachedout) [inline salt/modules/state.py]: `TODO`

File: salt/modules/zypper.py [hunk 11]
```diff
@@ -32,7 +32,7 @@
 from xml.parsers.expat import ExpatError
 
 # Import salt libs
-import salt.utils  # Can be removed once alias_function, is_true are moved
+import salt.utils  # Can be removed once alias_function is moved
```
Comment (cachedout) [inline salt/modules/zypper.py]: `TODO`

**Review by cachedout — CHANGES_REQUESTED** [review]

**Comment by terminalmage** [comment]
@cachedout I've changed all "Can be removed" lines into TODOs.

**Comment by terminalmage** [comment]
Some legit failures here, I will address these on Friday.

**Comment by terminalmage** [comment]
Tests should pass now.

File: tests/integration/modules/test_key.py [hunk 12]
```diff
@@ -14,6 +14,7 @@ def test_key_finger(self):
         test key.finger to ensure we receive a valid fingerprint
         '''
         out = self.run_function('key.finger')
+        print(out)
```
Comment (rallytime) [inline tests/integration/modules/test_key.py]: I think you left a debugging line in here. Can you remove this?

**Review by rallytime — CHANGES_REQUESTED** [review]
Just one small comment. Otherwise, this looks good.

File: tests/integration/modules/test_key.py (hunk 12 above)
Comment (terminalmage) [reply tests/integration/modules/test_key.py]: Done.

**Review by rallytime — APPROVED** [review]

**Comment by terminalmage** [comment]
re-run lint

**Comment by rallytime** [comment]
Test failures are unrelated.

**Pull request merged**